fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/provider/foundationdb/OnlineIndexerMutualTest.java

```diff
@@ -451,6 +455,132 @@ void testMutualIndexingCrashAndAllowContinueNonMutually(boolean explicit) {
         assertAllValidated(indexes);
     }
 
+    @ParameterizedTest
```

Comment:
It seems like there should be a test where there are two indexers,
1. building `MULTI_TARGET_BY_RECORDS` and has `MUTUAL_TO_SINGLE` allowed
2. building `MUTUAL_BY_RECORDS` and has `BY_RECORDS_TO_MUTUAL` allowed

I believe the two will both do indexing, with the state constantly swapping.
If that is the case, probably worth calling out in the documentation, because you basically have no isolation between the two, and would presumably get a lot of conflicts if you had the two running.

Reply:
(Did you mean the first item to be 
1. building `BY_RECORDS` and has `MUTUAL_TO_SINGLE` allowed
?)